test/data-plane/library.sh

```diff
@@ -170,10 +170,7 @@ function k8s() {
 
 function data_plane_unit_tests() {
 
-  docker build \
-    --file ${DATA_PLANE_DIR}/docker/test/Dockerfile \
-    --build-arg JAVA_IMAGE=${JAVA_IMAGE} \
-    --tag tests ${DATA_PLANE_DIR}
+  java_ut || java_ut || fail_test "Data plane unit tests failed"
```

Comment:
Are you retrying unit test 2 times?

Reply:
I was expecting this PR just retries building 2 times

Reply:
Our running Java unit tests meaning is building the test container image.

Reply:
I guess we can't skip atm the 2 phases right?

Reply:
Yes, that's right. :(

Reply:
Well, that's an issue of prow itself, so who cares :smile: